nautobot_data_validation_engine/api/serializers.py

```diff
@@ -66,3 +79,56 @@ class Meta:
             "created",
             "last_updated",
         ]
+
+
+class RequiredValidationRuleSerializer(NautobotModelSerializer):
+    """Serializer for `RequiredValidationRule` objects."""
+
+    url = serializers.HyperlinkedIdentityField(
+        view_name="plugins-api:nautobot_data_validation_engine-api:requiredvalidationrule-detail"
+    )
+    content_type = ContentTypeField(
+        queryset=ContentType.objects.filter(FeatureQuery("custom_validators").get_query()),
+    )
+
+    class Meta:
+        model = RequiredValidationRule
+        fields = [
+            "id",
+            "url",
+            "name",
+            "slug",
+            "content_type",
+            "field",
+            "enabled",
+            "error_message",
+            "created",
+            "last_updated",
+        ]
```

Comment:
Note that when using `NautobotModelSerializer`, the `id`, `display`, `created`, and `last_updated` fields are automatically added to `fields` list, so you don't *need* to explicitly specify them (though it's not harmful to do so).